docs/dataformats/observation_lists.rst

```diff
@@ -10,38 +10,87 @@ the information in the run list. These filenames can either be given directly
 or there can be naming conventions, e.g. the event list for run ``42`` could be stored
 at ``$TEV_DATA/RUN_000042/Events.fits``.
 
+A run list must have at least a column called ``OBS_ID``::
+ 
+   OBS_ID
+   42
+   43
 
-CSV format
-----------
+Usually it has many more columns with information about each observation. A list of Gammapy supported columns is:
 
-We use the `CSV <http://en.wikipedia.org/wiki/Comma-separated_values>`_ (comma-separated values) format for run lists.
-This has the advantage that everyone can work with whatever tool they like. Here's some good options:
+================  ================================================================================================  =========
+column name       description                                                                                       required?
+================  ================================================================================================  =========
+OBS_ID            observation ID as an integer                                                                      yes
+RA                pointing position right ascension in equatorial (ICRS) coordinates                                yes?
+DEC               pointing position declination in equatorial (ICRS) coordinates                                    yes?
+AZ                average azimuth angle during the observation                                                      yes?
+ALT               average altitude angle during the observation                                                     yes?
+MUON_EFFICIENCY   average muon efficiency of the telescopes                                                         yes?
+TIME_START        start time of the observation stored as number of seconds after the time reference in the header  yes?
```

Comment:
Ok, done.
Does this mean we should use the same time reference as Fermi?
http://fermi.gsfc.nasa.gov/ssc/data/analysis/documentation/Cicerone/Cicerone_Data/Time_in_ScienceTools.html
ref: midnight at the beginning of January 1, 2001, in the UTC system
I think that "mission elapsed time" (MET) is a bit confusing, since I would imagine each experiment would have a different reference, depending on the starting date. Or does this refer to a well established system, where everyone uses the above defined time reference?